test/controllers/AddAdditionalMetadataControllerSpec.scala

```diff
@@ -245,6 +245,21 @@ class AddAdditionalMetadataControllerSpec extends FrontEndTestHelper {
       playStatus(addAdditionalMetadataPage) mustBe FOUND
     }
 
+    "return a redirect to the additionalMetadataClosureStatus page if the closure status isn't closed" in {
```

Comment:
Could you add another specific test for the correct redirect when a file is `closed`. It just makes sure that this is tested independently, instead of being indirectly checked in other unit tests.